weather_mv/loader_pipeline/bq.py

```diff
@@ -174,6 +194,49 @@ def validate_arguments(cls, known_args: argparse.Namespace, pipeline_args: t.Lis
                             region=pipeline_options_dict.get('region'))
             logger.info('Region validation completed successfully.')
 
+    def generate_csv(
+        self,
+        csv_path: str,
+        lats: t.List,
+        lons: t.List,
+        lat_grid_resolution: float,
+        lon_grid_resolution: float,
+        skip_creating_polygon: bool = False,
+    ):
+        """Generates geo data CSV."""
```

Comment:
What do you think about using parquet, or geoparquet here instead of a CVS?

Reply:
Thanks Alex, this make sense. Moved to parquet; file size reduced to 91 MiB compare to 890 MiB.